.kitchen.cloud.yml

```diff
@@ -1,72 +1,28 @@
 ---
 driver_config:
-  aws_access_key_id: <%= ENV['AWS_ACCESS_KEY_ID'] %>
-  aws_secret_access_key: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
-  aws_ssh_key_id: <%= ENV['AWS_KEYPAIR_NAME'] %>
-  availability_zone: <%= ENV['AWS_AVAILABILITY_ZONE'] %>
+  iam_profile_name: ChefKitchenTestAws
+  aws_ssh_key_id: chef-kitchen-test-aws
```

Comment:
This file needs to get updated with the contents of the latest file in master.  I've updated all the boxes to the latest released and made sure we test on RHEL platforms as well.  Also we need to make sure that any driver config is configurable and works for any users.  We need to be able to set these in travis at some point so we can test on each PR.